code/game/objects/items/cardboard_cutouts.dm

```diff
@@ -0,0 +1,175 @@
+	else if(I.hitsound)
+		playsound(loc, I.hitsound, 20, 1, -1)
+
+	user.changeNext_move(CLICK_CD_MELEE)
+	user.do_attack_animation(src)
+
+	if(I.force)
+		user.visible_message("<span class='danger'>[user] has hit \
+			[src] with [I]!</span>", "<span class='danger'>You hit [src] \
+			with [I]!</span>")
+
+		if(prob(I.force))
+			push_over()
+
+/obj/item/cardboard_cutout/bullet_act(obj/item/projectile/P)
+	visible_message("<span class='danger'>[src] has been hit by [P]!</span>")
+	playsound(src, 'sound/weapons/slice.ogg', 50, 1)
+	if(prob(P.damage))
+		push_over()
+
+/obj/item/cardboard_cutout/proc/change_appearance(obj/item/toy/crayon/crayon, mob/living/user)
+	if(!crayon || !user)
+		return
+	if(pushed_over)
+		to_chat(user, "<span class='warning'>Right [src] first!</span>")
+		return
+	var/new_appearance = input(user, "Choose a new appearance for [src].", "26th Century Deception") as null|anything in possible_appearances
+	if(!new_appearance || !crayon)
+		return
+	if(!do_after(user, 10, FALSE, src, TRUE))
+		return
+	user.visible_message("<span class='notice'>[user] gives [src] a new look.</span>", "<span class='notice'>Voila! You give [src] a new look.</span>")
+	alpha = 255
+	icon = initial(icon)
+	if(!deceptive)
+		color = "#FFD7A7"
+	switch(new_appearance)
+		if("Assistant")
+			name = "[pick(first_names_male)] [pick(last_names)]"
+			desc = "A cardboard cutout of an assistant."
+			icon_state = "cutout_greytide"
+		if("Clown")
+			name = pick(clown_names)
+			desc = "A cardboard cutout of a clown. You get the feeling that it should be in a corner."
+			icon_state = "cutout_clown"
+		if("Mime")
+			name = pick(mime_names)
+			desc = "...(A cardboard cutout of a mime.)"
+			icon_state = "cutout_mime"
+		if("Traitor")
+			name = "[pick("Unknown", "Captain")]"
+			desc = "A cardboard cutout of a traitor."
+			icon_state = "cutout_traitor"
+		if("Nuke Op")
+			name = "[pick("Unknown", "COMMS", "Telecomms", "AI", "stealthy op", "STEALTH", "sneakybeaky", "MEDIC", "Medic")]"
+			desc = "A cardboard cutout of a nuclear operative."
+			icon_state = "cutout_fluke"
+		if("Cultist")
+			name = "Unknown"
+			desc = "A cardboard cutout of a cultist."
+			icon_state = "cutout_cultist"
+		if("Clockwork Cultist")
+			name = "[random_name(pick(MALE,FEMALE))]"
+			desc = "A cardboard cutout of a servant of Ratvar."
+			icon_state = "cutout_servant"
+		if("Revolutionary")
+			name = "Unknown"
+			desc = "A cardboard cutout of a revolutionary."
+			icon_state = "cutout_viva"
+		if("Wizard")
+			name = "[pick(wizard_first)], [pick(wizard_second)]"
+			desc = "A cardboard cutout of a wizard."
+			icon_state = "cutout_wizard"
+		if("Shadowling")
+			name = "Unknown"
+			desc = "A cardboard cutout of a shadowling."
+			icon_state = "cutout_shadowling"
+		if("Xenomorph")
+			name = "alien hunter ([rand(1, 999)])"
+			desc = "A cardboard cutout of a xenomorph."
+			icon_state = "cutout_fukken_xeno"
+			if(prob(25))
+				alpha = 75 //Spooky sneaking!
```

Comment:
What's the purpose of adding a 25% chance of making the sprite transparent? Why not just add another cutout option for stealthed xenos?